Tests/Core/NetworkObserverTests.swift

```diff
@@ -80,4 +80,63 @@ class NetworkObserverTests: OperationTests {
             XCTAssertFalse(self.visibilityChanges[1])
         }
     }
+
+    func test__network_indicator_changes_once_when_multiple_operations_start() {
+
+        let operation1 = TestOperation(delay: 1)
+        operation1.addObserver(NetworkObserver(indicator: indicator))
+
+        let operation2 = TestOperation(delay: 1)
+        operation2.addObserver(NetworkObserver(indicator: indicator))
+
+        addCompletionBlockToTestOperation(operation1, withExpectation: expectationWithDescription("Test: \(#function)"))
+        addCompletionBlockToTestOperation(operation2, withExpectation: expectationWithDescription("Test: \(#function)"))
```

Comment:
Hi @ryanjm - sorry for the delay getting to this....

The whole `addCompletionBlockToTestOperation...` stuff is just some helper methods in the `OperationTest` base class. And it helps setup the expectation fulfilment for the async testing. However, this has evolved over the course of the project (and I should really clean everything up and unify it all). Now, you can just do something like this:

```
func test__network_indicator_changes_once_when_multiple_operations_start() {

    let operation1 = TestOperation(delay: 1)
    operation1.addObserver(NetworkObserver(indicator: indicator))

    let operation2 = TestOperation(delay: 1)
    operation2.addObserver(NetworkObserver(indicator: indicator))

    waitForOperations(operation1, operation2)

    XCTAssertTrue(operation1.didExecute)
    XCTAssertTrue(operation1.finished)
    XCTAssertTrue(operation2.didExecute)
    XCTAssertTrue(operation2.finished)
    XCTAssertTrue(self.visibilityChanges[0])
    XCTAssertEqual(self.visibilityChanges.count, 1)
}